src/cli/organize.py

```
@@ -16,6 +17,66 @@
+       server-side with rate limits.
+    """
+    cpu = os.cpu_count() or 1
+    cap = min(_AUTO_WORKERS_CEILING, max(1, cpu // 2))
+
+    # Lazy-import to avoid pulling provider_env (and its loguru chain)
+    # at CLI startup for `fo version` etc. (#404 fast-path).
+    try:
+        from config.provider_env import get_current_provider
+
+        provider = get_current_provider()
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)